docs/docs/reference/release-notes/v2-maintenance-plan.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gatsby Framework Version Support
+---
+
+## Overview
+
+As we continue evolving the Gatsby framework, and as we move from one major version to the next, we will communicate the duration and types of support provided to the prior major version. This allows users to effectively plan their Gatsby framework upgrades.
+
+## What is happening, when?
+
+Gatsby Version 2.32 will enter Maintenance Long Term Support on **June 8, 2021**. At that same time, Gatsby Version 3 will be considered the Active Long Term Support version of Gatsby. The Maintenance Long Term Support will continue until the release of Gatsby v4 (date TBD).
+
+## What does this mean to users?
+
+- As of June 8, 2021, Gatsby version 2.32 will only receive critical patches.
+- To take advantage of new capabilities within Gatsby, you must be on version 3.0 or later
```

Comment:
This could be a good place to link to the v3 migration guide

Reply:
Good call - added.